website/components/Plugins.tsx

```diff
@@ -6,8 +6,9 @@ export const Plugins = () => {
     <div className="grid grid-cols-1 mt-12 gap-x-6 gap-y-12 sm:grid-cols-2 lg:mt-16 lg:grid-cols-3 lg:gap-x-8 lg:gap-y-12">
       {INTEGRATIONS.map(({ name, id, logo }) => (
         <a
-          href={`/plugins/${id}`}
+          href={`https://github.com/cloudquery/cloudquery/tree/main/plugins/source/${id}`}
```

Comment:
```suggestion
          href={`https://github.com/cloudquery/cloudquery/tree/${process.env.VERCEL_GIT_COMMIT_REF || `main`}/plugins/source/${id}`}
```

Not sure about the syntax but this ensures this works for deploy previews too. `VERCEL_GIT_COMMIT_REF` is the PR branch